src/scripts/main.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+const list = document.getElementById('thumbs');
+const bigItem = document.getElementById('largeImg');
+
+list.addEventListener('click', (event) => {
+  event.preventDefault();
+
+  const currentItemSrc = event.target.src;
```

Comment:
Pay attention to the index.html. You should take the href of `a` tag, not src of thumb to improve the quality of images
![image](https://github.com/mate-academy/js_gallery_DOM/assets/96409563/3b59127d-f2b9-49d6-8fc7-bc534453159d)